src/main/webapp/search.jsp

```diff
@@ -4,6 +4,9 @@
 <t:genericpage>
     <jsp:body>
       <div class="content">
+          <script type="text/javascript">
+            checkIngredientsList();
```

Comment:
What is this function doing?

Reply:
This function was used when I had 2 different inputs for the title and ingredients, I forgot to delete it in the last commit.